study.md

```diff
@@ -22,4 +22,22 @@ should have something to contribute.
 
 ## Summary
 
-<!-- your notes here -->
+NoSQL is a non-relational database which is made to store complex types of data
+in which the relational database is difficult to do.
+
+### Identify use cases where relational or non-relational databases are appropriate.
+If the relationship of data is not neccessary and important for a case, then using
+non-relational databases is more flexibility for complex data types and efficiency.
+
+If the relationship of data is important and complex, such as social data , then using
+relationship databases is neccessary.
+
+### List three strengths of relational databases
+- strong consistency
+- spread databases over many servers
```

Comment:
If this means the same database spread over several servers, then it should be in the NoSQL database section.